yt_dlp/extractor/goodgame.py

```diff
@@ -17,26 +18,26 @@ class GoodGameIE(InfoExtractor):
             'title': r're:Рейтинговые герои \d{4}-\d{2}-\d{2} \d{2}:\d{2}$',
             'description': 'md5:ca5faa69b5d9215afdc65caeca68e1f6',
             'thumbnail': r're:^https?://.*\.jpg$',
-            'is_live': True,
```

Comment:
@pukkandan `live_status` takes precedence over `is_live` property?

Reply:
`is_live` is not needed in tests